rd_ui/app/views/users/show.html

```diff
@@ -24,9 +24,22 @@ <h3 class="p-l-5">{{user.name}}</h3>
     </p>
     <div ng-if="currentUser.isAdmin">
       <hr/>
-      <button class="btn btn-default" ng-click="sendPasswordReset()" ng-disabled="disablePasswordResetButton">Send
-        Password Reset Email
-      </button>
+      <div class="form-group">
+        <button class="btn btn-default" ng-click="sendPasswordReset()" ng-disabled="disablePasswordResetButton">Send
+          Password Reset Email
+        </button>
+      </div>
+
+      <div ng-if="passwordResetLink && !disablePasswordResetButton" class="alert alert-success">
+        <p>
```

Comment:
Oh, I see. Just changed it. To be honest, as a user, I'd prefer to have the link shown regardless if I have an e-mail server configured or not, similar to the user invites. I find them useful even when the user will receive an e-mail, and it also gives admins the ability to change a user's password herself. Happy to go with whatever you think is best, though.

Just to let you know, I'm adding new commits and not changing the original ones just because I think it's easier to review this way, but I expect that you (or me) will squash them before merging.